CODEOWNERS

```diff
@@ -0,0 +1,115 @@
+/task/show-sbom-rhdh                        @konflux-ci/build-maintainers
+/task/slack-webhook-notification            @konflux-ci/build-maintainers
+/task/source-build                          @konflux-ci/build-maintainers
+/task/source-build-oci-ta                   @konflux-ci/build-maintainers
+/task/summary                               @konflux-ci/build-maintainers
+/task/update-deployment                     @konflux-ci/build-maintainers
+/task/update-infra-deployments              @konflux-ci/build-maintainers
+/task/upload-sbom-to-trustification         @konflux-ci/build-maintainers
+
+# renovate groupName=build
+/task/prefetch-dependencies         @konflux-ci/build-maintainers @brunoapimentel @eskultety @taylormadore
+/task/prefetch-dependencies-oci-ta  @konflux-ci/build-maintainers @brunoapimentel @eskultety @taylormadore
+
+# renovate groupName=build
+/task/generate-labels   @konflux-ci/build-maintainers @ralphbean
+
+# renovate groupName=ec
+/task/tkn-bundle                    @konflux-ci/ec
+/task/tkn-bundle-oci-ta             @konflux-ci/ec
+/task/verify-enterprise-contract    @konflux-ci/ec
+
+# renovate groupName=integration
+/task/clair-scan                @konflux-ci/integration-service-maintainers
+/task/clamav-scan               @konflux-ci/integration-service-maintainers
+/task/deprecated-image-check    @konflux-ci/integration-service-maintainers
+/task/fbc-related-image-check   @konflux-ci/integration-service-maintainers
+/task/fbc-validation            @konflux-ci/integration-service-maintainers
+/task/inspect-image             @konflux-ci/integration-service-maintainers
+/task/sast-snyk-check-oci-ta    @konflux-ci/integration-service-maintainers
+/task/sast-unicode-check-oci-ta @konflux-ci/integration-service-maintainers
+/task/sbom-json-check           @konflux-ci/integration-service-maintainers
+/task/validate-fbc              @konflux-ci/integration-service-maintainers
+
+# renovate groupName=integration
+/task/coverity-availability-check           @konflux-ci/integration-service-maintainers @kdudka
+/task/coverity-availability-check-oci-ta    @konflux-ci/integration-service-maintainers @kdudka
+/task/sast-coverity-check                   @konflux-ci/integration-service-maintainers @kdudka
+/task/sast-coverity-check-oci-ta            @konflux-ci/integration-service-maintainers @kdudka
+/task/sast-shell-check                      @konflux-ci/integration-service-maintainers @kdudka
+/task/sast-shell-check-oci-ta               @konflux-ci/integration-service-maintainers @kdudka
+/task/sast-snyk-check                       @konflux-ci/integration-service-maintainers @kdudka
+/task/sast-unicode-check                    @konflux-ci/integration-service-maintainers @kdudka
+
+# renovate groupName=preflight
+/task/ecosystem-cert-preflight-checks   @acornett21 @bcrochet @komish @skattoju
+
+# renovate groupName=eaas
+/task/provision-env-with-ephemeral-namespace    @amisstea @avi-biton @gbenhaim @oamsalem @yftacherzog
```

Comment:
```suggestion
/task/provision-env-with-ephemeral-namespace    @amisstea @avi-biton @gbenhaim @omeramsc @yftacherzog
```

Reply:
:scream: good catch. @arewm we may have invited the wrong Omar into konflux-ci

Reply:
Fixed. Went through the rest of the list, I think the rest are legit